src/stores/main/selectors.ts

```diff
@@ -86,15 +87,60 @@ export function getLoggedInUser(state: IState): ILoggedInUser | null {
   return state.main.loggedInUser;
 }
 
-export function getHasAccessToResources(state: IState): boolean {
+export function selectHasPermission(
```

Comment:
I can't find where `selectHasPermission` used. Am I missing something?

Reply:
It's not actively used in this PR, but it's a must for any other feature. We need to be able to easily check if a user has the required permission to use a feature on the UI side.

Here's an example of how it can be used (with `selectHasPermission` instead of `getUserIsAdmin`):
https://github.com/giantswarm/happa/blob/dbe7906d471fe0add5eb90083e1e0503e666825d/src/components/Apps/AppsList/AppsList.tsx#L60

It's just a wrapper over `hasPermissions` that can be used easier as a Redux (global state) selector.

Reply:
We talked, I know a bit more now :)